extensions/sql-migration/src/constants/strings.ts

```diff
@@ -337,6 +337,7 @@ export function LOGIN_MIGRATIONS_GET_LOGINS_ERROR(message: string): string {
 	return localize('sql.migration.wizard.target.login.error', "Error getting login information: {0}", message);
 }
 export const SELECT_LOGIN_TO_CONTINUE = localize('sql.migration.select.database.to.continue', "Please select 1 or more logins for migration");
+export const ENTER_AAD_DOMAIN_NAME = localize('sql.login.migration.enter.AAD.domain.name.to.continue', "AAD Domain name is required to migrate a windows login. Please enter an AAD Domain Name or deselect windows login(s).");
```

Comment:
nit: Windows instead of windows

Reply:
also perhaps use the full form of AAD?

Reply:
ahh didn't see the second comment. will update